### PR TITLE
Add cache timeout test

### PR DESCRIPTION
**Pull request by sethrosetter**
- Add 1k users 5k docs channel cache overload test
- Simplify server provisioning
- Improved test output for changes request sanity checks
- Minor ansible tweaks to sg installation
- _changes request adds ability for filters
- docs id verifier for changes feed

### CODE REVIEW

File: lib/user.py [hunk 1]
```diff
@@ -8,9 +8,11 @@
 import base64
 import uuid
 import re
+from lib.document import Document
 from collections import defaultdict
 from responseprinter import ResponsePrinter
 
+PRINTER = ResponsePrinter()
```
Comment (tleyden) [inline lib/user.py]: What about `response_printer` instead of `PRINTER`?

Comment (sethrosetter) [inline lib/user.py]: I was just emphasizing that it is global and will be shared among threads but if that is the convention for globals, I'm in agreement.

Comment (tleyden) [inline lib/user.py]: I don't see anything about using all caps here:  https://www.python.org/dev/peps/pep-0008/#id35

Comment (sethrosetter) [inline lib/user.py]: @tleyden fixed. Also changed the name as it is more than a "response" printer now

**Comment by tleyden** [comment]
LGTM -- go ahead and merge from my POV (it's assigned to @ashvindersingh so I'm holding off on merging)

**Comment by sethrosetter** [comment]
@tleyden Thanks. I will was just looking for a quick once over in ansible land. But I appreciate the overall feedback!

**Pull request merged**